watertap/data/techno_economic/pump.yaml

```diff
@@ -0,0 +1,115 @@
+default:
```

Comment:
This can be for a subsequent PR, but @kurbansitterley pointed out that there are several other yaml files (without the word "pump" in them) that are actually supposed to contain parameters for different pumps.

Reply:
I would need a list of those files to know what to include. @kurbansitterley could you provide a list of any other data files for pumps that should be included here?

Reply:
From a costing perspective, in addition to anything with "pump" in the name, these are all pumps:

- agriculture
- discharge
- industrial
- irrigation
- mining

Reply:
@kurbansitterley Do you have suggestions for better names for those sub-types? "agriculature" is not really a sub-type for a pump (hence why I missed it in the first pass).

Reply:
Even more so - all those subtypes use the same values so we should have a general name that covers all those cases.

Reply:
These are all identical in costing to the separate water_pumping_station module, so you could just use that. Or, something like "use_pumping_station" since these were all the last units in the treatment train to pump the water to their end use (i.e. for agriculture or mining).